dtfs-central-api/src/v1/controllers/tfm/deal/tfm-get-deals.controller.js

```diff
@@ -2,11 +2,27 @@ const moment = require('moment');
 const db = require('../../../../drivers/db-client');
 const CONSTANTS = require('../../../../constants');
 const getObjectPropertyValueFromStringPath = require('../../../../utils/getObjectPropertyValueFromStringPath');
+const getCrossDataModelProperty = require('../../../../mapping/getCrossDataModelProperty');
+
+const isEmpty = (value) => {
+  if (value === null) return '';
+  return value;
+};
 
 const sortDeals = (deals, sortBy) =>
   deals.sort((xDeal, yDeal) => {
-    const xField = getObjectPropertyValueFromStringPath(xDeal, sortBy.field);
-    const yField = getObjectPropertyValueFromStringPath(yDeal, sortBy.field);
+    const xField = isEmpty(
+      getObjectPropertyValueFromStringPath(
+        xDeal,
+        getCrossDataModelProperty(xDeal, sortBy.field),
+      ),
+    );
+    const yField = isEmpty(
+      getObjectPropertyValueFromStringPath(
+        yDeal,
+        getCrossDataModelProperty(yDeal, sortBy.field),
+      ),
+    );
```

Comment:
`isEmpty` feels strange to me because it returns a value, i'd expect it to return true or false - and could be extracted into utils/helper directory with tests, so can be used anywhere.

I'd do it like this:

```js
const xField = !isEmpty && (
  getObjectPropertyValueFromStringPath(
    xDeal,
    mapFieldName(xDeal, sortBy.field),
  ),
);
```

Also think `mapFieldName` is cleaner name than `getCrossDataModelProperty`.

What do you think?